test/Object/rlexe.xml

```diff
@@ -195,6 +195,12 @@
   </test>
   <test>
     <default>
+      <files>getOwnPropertyDescriptors.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
```

Comment:
Passing the flag in here is a good idea too - that way unit tests still pass if you mess with what is on by default.